script/generate-expressions.mjs

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs'
+import path from 'path'
+import regenerate from 'regenerate'
+import codes from '../lib/character/codes.js'
```

Comment:
Could you add `codes` below all `@unicode` imports? I guess I missed that before. But I prefer local files last.

Reply:
Sure, me too. Usually rely on eslint-plugin-import though hate it dependency tree